core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -1395,6 +1395,15 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.MASTER)
           .build();
+  public static final PropertyKey MASTER_UFS_BLOCK_LOCATION_ALL_FALLBACK_ENABLED =
```

Comment:
no. this parameter controls the behavior in 2 situations:
1. ufs is remote
2. ufs does not contain any block locations either. it may be local or remote

Reply:
updated the comment to make it not specific to remote ufs